.github/workflows/luacheck.yml

```diff
@@ -11,3 +11,11 @@ jobs:
         uses: actions/checkout@v3
       - name: Luacheck
         uses: lunarmodules/luacheck@v0
+
+      - name: Install EditorConfig Lint
+        uses: editorconfig-checker/action-editorconfig-checker@d4fca16fc71adef10fbe101903b654449fa9570c
+      - name: Check all files for linting issues (not fatally)
+        run: editorconfig-checker || true
```

Comment:
This needs to fail if it finds problems. We can mark the entire CI job as not required, but we do want it to throw the right status code.